pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +414,265 @@ def image_threshold(
+        Apply reverse FFT to an example image.
+
+        >>> from pyvista import examples
+        >>> image = examples.download_moonlanding_image()
+        >>> fft_image = image.fft()
+        >>> image_again = fft_image.rfft()
+        >>> image_again.point_data  # doctest:+SKIP
+        pyvista DataSetAttributes
+        Association     : POINT
+        Active Scalars  : PNGImage
+        Active Vectors  : None
+        Active Texture  : None
+        Active Normals  : None
+        Contains arrays :
+        PNGImage                float64  (298620, 2)          SCALARS
+
+        See :ref:`image_fft_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageRFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Reverse Fast Fourier Transform.')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def low_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name='ImageScalars',
+        progress_bar=False,
+    ):
+        """Perform a low pass filter in the frequency domain.
+
+        This filter only works on an image after it has been converted to
+        frequency domain by a :func:`UniformGridFilters.fft` filter.
+
+        A :func:`UniformGridFilters.rfft` filter can be
+        used to convert the output back into the spatial
+        domain. This filter attenuates high frequency components.
+        Input and output are in doubles, with two components.
+
+        Parameters
+        ----------
+        x_cutoff : double
+            The cutoff frequency for the x axis.
+
+        y_cutoff : double
+            The cutoff frequency for the y axis.
+
+        z_cutoff : double
+            The cutoff frequency for the z axis.
+
+        order : int, optional
+            The order of the cutoff curve. Given from the equation
+             ``(1 + pow(CutOff/Freq(i, j), 2*Order))``.
+
+        output_scalars_name : str, optional
+            The name of the output scalars. By default this is
+            ``'ImageScalars'``.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            UniformGrid with the applied low pass filter.
+
+        Examples
+        --------
+        See :ref:`image_fft_perlin_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageButterworthLowPass()
+        alg.SetInputDataObject(self)
+        alg.SetCutOff(x_cutoff, y_cutoff, z_cutoff)
+        alg.SetOrder(1)
+        _update_alg(alg, progress_bar, 'Performing Low Pass Filter')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def high_pass(
```

Comment:
See comments for `low_pass()`.

Reply:
Fixed in 2b26cccfc95ac76c5c082c82258e360aa64712d7